packages/editor/src/Relation/PanZoom/LinesDiagram/LinesDiagram.tsx

```diff
@@ -137,7 +137,9 @@ export const LinesDiagram = React.forwardRef<
         x: -nodeX * s + size.width / 2.0,
         y: -nodeY * s + size.height / 2.0,
       };
-      setTransform(transformTo.x, transformTo.y, s, 200, "easeOut");
+      setTimeout(() => {
+        setTransform(transformTo.x, transformTo.y, s, 200, "easeOut");
+      }, 250);
```

Comment:
To mi sie niezbyt podoba, ale nie chciało mi transformowac do dodanego noda bez tego. Może dlatego że setSelectedNode też jest z timeoutem
`const setSelectedNodeId = useCallback(
    (_selectedNodeId?: SelectedNodeId) => {
      const nodeId = _selectedNodeId?.value?.id;
      DOMEvents.selectNode.trigger(nodeId);
      if (
        nodeId !== selectedNodeId?.value?.id ||
        _selectedNodeId?.justCreated !== selectedNodeId?.justCreated
      ) {
        setTimeout(() => {
          return _setSelectedNodeId(_selectedNodeId);
        }, 250);
      }
    },
    [_setSelectedNodeId, allNodes, selectedNodeId?.value?.id, focusMode]
  );`
  
  Troche spowalnia to przesuwanie ale moim zdaniem bez uszczerbku na UI. Możemy o tym podyskutować ofc jak masz pomysl dlaczego tak jest. Bo to jakby chce dodac napierw node ktory nie jest aktualnie wyrenderowany a potem oznaczyc go selected

Reply:
nie no uszczerbek jest i to bardzo duży, pomyślimy co z tym zrobić

Reply:
ok wywaliłam oba timeouty na pierwszy rzut oka działa ok, ale nie wiem po so on był w tym setSelectedId może ty pamiętasz co trzeba potestowac żeby sprawdzic czy nic się ie popsuło

Reply:
przerzucam na wip jeszcze będę testować